views/dashboard-parts.hbs

```diff
@@ -29,47 +29,46 @@
         {{> navbar }}
 
         <div class='container-fluid checkout-body'>
-
+            <div align="center">
+				<img src="/img/parts.png" height="50" width="180">
+			</div>
             <div class='sidebar col-md-2'>
-                <p class='search-bar'>Search for a part:</p>
-                {{> filter}}
+                <br>
+                <p class='search-bar'>SEARCH FOR A PART:</p>
+                {{>filter}}
                 <p class='category-header'>Browse by category</p>
                 <p class='part-category'>ALL</p>
-                <p class='part-category'>Dev Kit</p>
-                <p class='part-category'>Battery</p>
-                <p class='part-category'>Connector</p>
-                <p class='part-category'>Display</p>
-                <p class='part-category'>Interface IC</p>
+                <p class='part-category'>Dev Kits</p>
+                <p class='part-category'>Batteries</p>
+                <p class='part-category'>Power</p>
+                <p class='part-category'>Microcontrollers</p>
+                <p class='part-category'>FPGAs</p>
                 <p class='part-category'>Sensors</p>
+                <p class='part-category'>Connectors</p>
+                <p class='part-category'>Displays</p>
+                <p class='part-category'>Interface IC</p>
                 <p class='part-category'>User Interface</p>
-                <p class='part-category'>Power</p>
-                <p class='part-category'>Wireless</p>
-                <p class='part-category'>Discrete Semiconductor</p>
+                <p class='part-category'>Discrete Semiconductors</p>
```

Comment:
I know it wasn't made clear in comments or anything, but the reason these categories are listed as singular rather than plural is actually because they specifically correspond to values associated with each item which allows us to filter so we can't change the displayed categories without directly changing the backend as well.